tests/utils/utils.test.js

```
@@ -1,49 +1,49 @@
 
-	it("should return false", function() {
+	it("should return false", function () {
 		const result = ParseBoolean("false");
 		expect(result).to.be.false;
 	});
 
-	it("should return false", function() {
+	it("should return false", function () {
 		const result = ParseBoolean(null);
 		expect(result).to.be.false;
 	});
```

ESLint

[error] Test title is used multiple times in the same test suite.

(mocha/no-identical-title)